sys/riscv/riscv/pmap.c

```diff
@@ -3483,6 +3531,9 @@ pmap_copy_page(vm_page_t msrc, vm_page_t mdst)
 void
 pmap_copy_page_tags(vm_page_t msrc, vm_page_t mdst)
 {
+
+	VM_PAGE_ASSERT_PGA_CAPMETA_COPY(msrc, mdst);
```

Comment:
FreeBSD's style still doesn't really define variables in the middle of scopes.  I would perhaps move the assertion just before the call to `pagecopy`.